migration_original/ODS1Stage/tables/Mid/Facility/spu_translated_Facility.sql

```diff
@@ -0,0 +1,1586 @@
+    search.searchtypedescription as facilitysearchtype,
+    f.accreditation,
+    f.accreditationdescription,
+    f.treatmentschedules,
+    fad.phonenumber,
+    f.additionaltransportationinformation,
+    f.afterhoursphonenumber,
+    f.awardsinformation,
+    f.closedholidaysinformation,
+    f.communityactivitiesinformation,
+    f.communityoutreachprograminformation,
+    f.communitysupportinformation,
+    f.emergencyafterhoursphonenumber,
+    f.facilitydescription,
+    f.foundationinformation,
+    f.healthplaninformation,
+    f.ismedicaidaccepted,
+    f.ismedicareaccepted,
+    f.isteaching,
+    f.languageinformation,
+    f.medicalservicesinformation,
+    f.missionstatement,
+    f.officeclosetime,
+    f.officeopentime,
+    f.onsiteguestservicesinformation,
+    f.othereducationandtraininginformation,
+    f.otherservicesinformation,
+    f.ownershiptype,
+    f.parkinginstructionsinformation,
+    f.paymentpolicyinformation,
+    f.professionalaffiliationinformation,
+    f.publictransportationinformation,
+    f.regionalrelationshipinformation,
+    f.religiousaffiliationinformation,
+    f.specialprogramsinformation,
+    f.surroundingareainformation,
+    case when f.facilityid = 'HGSTE6064176190065' then left(f.teachingprogramsinformation, 2997) else f.teachingprogramsinformation end as teachingprogramsinformation,
+    f.tollfreephonenumber,
+    f.transplantcapabilitiesinformation,
+    f.visitinghoursinformation,
+    f.volunteerinformation,
+    f.yearestablished,
+    f.hospitalaffiliationinformation,
+    f.physiciancallcenterphonenumber,
+    fr.ratingstar as overallhospitalstar,
+    traumalevel.adulttraumalevel,
+    traumalevel.pediatrictraumalevel,
+    case when hos_details.respgmapprama = 'Y' then 'AMA' end as respgmapprama, 
+    case when hos_details.respgmappraoa = 'Y' then 'AOA' end as respgmappraoa,
+    case when hos_details.respgmapprada = 'Y' then 'ADA' end as respgmapprada,
+    f.miscellaneousinformation,
+    f.appointmentinformation,
+    hos_details.url as website,
+    f.visitinghoursmonday,
+    f.visitinghourstuesday,
+    f.visitinghourswednesday,
+    f.visitinghoursthursday,
+    f.visitinghoursfriday,
+    f.visitinghourssaturday,
+    f.visitinghourssunday,
+    fac_details.facilityimagepath,
+    client_details.clienttoproductid,
+    client_details.clientcode,
+    client_details.clientname,
+    client_details.productcode,
+    client_details.productgroupcode,
+    -- phonexml
+    case when (ifnull(to_varchar(des.ClientToProductID),'') <> '') and exists 
+        (select TOP 1 DesignatedProviderPhone 
+        from Base.vwuPDCFacilityDetail fa 
+        where fa.PhoneTypeCode in ('PTUFS', 'PTHFS') 
+            and client_details.ClientProductToEntityID = fa.ClientProductToEntityID) then 
+        pfxml.phonexml 
+        when (select top 1 1 from Base.vwuPDCClientDetail cl where cl.PhoneTypeCode = 'PTHOS' and client_details.ClientToProductID = cl.ClientToProductID) is not null then 
```

Comment:
This could be limit 1
```suggestion
        when (select 1 from Base.vwuPDCClientDetail cl where cl.PhoneTypeCode = 'PTHOS' and client_details.ClientToProductID = cl.ClientToProductID LIMIT 1) is not null then 
```